### PR TITLE
update suggested to institute-of-physics-harvard to include report numbers

### PR DESCRIPTION
**Pull request by massimopinto**
[only one commit made, commit message copied here]
Using the current IOP-harvard dependent style
http://www.zotero.org/styles/physics-in-medicine-and-biology, report
numbers are not shown in report-type bibliography items. However,
report-type bibliography items are listed with report numbers in the
bibliography of articles appearing in the journal Physics in Medicine
and Biology. This, probably, is done manually by the user, by adding
the report number in the title field of a bibliography database (such
as Papers 3 for Mac).
This proposed update makes explicit use of the report number field and
formats it in Roman numerals, as at the format guidelines of IOP for
these type of references.
Example
Andreo P, Burns D T, Hohlfeld K, Huq M, Kanai T, Laitano R F, Smyth V
and Vynckier S 2000 Absorbed Dose Determination in External Beam
Radiotherapy, An International Code of Practice for Dosimetry Based on
Standards of Absorbed Dose to Water Report TRS-398, (Vienna:
International Atomic Energy Agency)

### CODE REVIEW

**Comment by rmzelle** [comment]
> This proposed update makes explicit use of the report number field and
formats it in Roman numerals, as at the format guidelines of IOP for
these type of references.
Example
Andreo P, Burns D T, Hohlfeld K, Huq M, Kanai T, Laitano R F, Smyth V
and Vynckier S 2000 Absorbed Dose Determination in External Beam
Radiotherapy, An International Code of Practice for Dosimetry Based on
Standards of Absorbed Dose to Water Report TRS-398, (Vienna:
International Atomic Energy Agency)

I checked http://authors.iop.org/atom/help.nsf/7765E7F03B07DE5080257020002E4D9B/$File/IOPWordGuidelines.pdf (the documentation link in this style), but I couldn't find any example of a reference for a report that includes "Report". Where did you find your example?

(also, presumably you meant "Roman type" text (i.e. non-italic), instead of "Roman numerals (which are numbers like "XVII"))

**Comment by massimopinto** [comment]
Scientific articles published in *Physics in Medicine and Biology* list the above mentioned technical report - which I will keep as an example - in a number of different ways in the Bibliography. I will enumerate just a few examples here.

1) **In** Adolfsson E, White S, Landry G, Lund E, Gustafsson H, Verhaegen F, Reniers B, Tedgren Å C and Carlsson G A 2015 Measurement of absorbed dose to water around an electronic brachytherapy source. Comparison of two dosimetry systems: lithium formate EPR dosimeters and radiochromic EBT2 film Phys Med Biol 3869–82
it appears as:
Andreo P *et al* 2000 *Absorbed Dose Determination in External Beam Radiotherapy, IAEA TRS 398:
an International Code of Practice for Dosimetry Based on Standards of Absorbed Dose to Water
(IAEA Technical Report Series vol 398)* (Vienna: International Atomic Energy Agency)
> that is, the report number is italicized and shown as "vol" which is correct for books but not for reports.

2) **In** Andreo P 2015 Dose to “water-like” media or dose to tissue in MV photons radiotherapy treatment planning: still a matter of debate Phys Med Biol 60 309–37
it appears as 
Andreo P, Burns D T, Hohlfeld K, Huq M S, Kanai T, Laitano F, Smyth V G and Vynckier S 2000
*Absorbed Dose Determination in External Beam Radiotherapy: an International Code of Practice for Dosimetry Based on Standards of Absorbed Dose-to-Water IAEA Technical Reports Series* vol 398) (Vienna: International Atomic Energy Agency)
> please note the vol 398 which is in roman type, "escaping" the title font, and prefixes a "vol" again.

3) **in** Hill R, Healy B, Holloway L, Kuncic Z, Thwaites D and Baldock C 2014 Advances in kilovoltage x-ray beam dosimetry Phys Med Biol 59 R183–R231
it appears as 
Andreo P, Burns D T, Hohlfield K, Huq M S, Kanai T, Laitano F, Smyth V and Vynckier S 2000
Absorbed dose determination in external beam radiotherapy, an international code of practice for dosimetry based on standards of absorbed dose to water *Technical Report Series No* 398 (Vienna: IAEA)
> not clear here why the title was not shown in Italic but the Technical Report..was

4) **in** Andreo P, Wulff J, Burns D T and Palmans H 2013 Consistency in reference radiotherapy dosimetry: resolution of an apparent conundrum when 60Co is the reference quality for charged-particle and photon beams Phys Med Biol 58 6593–621
it appears as
Andreo P, Burns D T, Hohlfield K, Huq M S, Kanai T, Laitano F, Smyth V G and Vynckier S 2000 Absorbed dose determination in external beam radiotherapy: an International Code of Practice for dosimetry based on standards of absorbed dose to water *IAEA Technical Report Series No. 398* (Vienna: International Atomic Energy Agency)

> As (3) above, but this time the report number is also in italic font.

5) **In** Krauss A, Büermann L, Kramer H-M and Selbach H-J 2012 Calorimetric determination of the absorbed dose to water for medium-energy x-rays with generating voltages from 70 to 280 kV Phys Med Biol 57 6245–68
it appears as
Andreo P, Burns D T, Hohlfeld K, Huq M S, Kanai T, Laitano F, Smyth V G and Vynckier S 2000 Absorbed dose determination in external beam radiotherapy: an international Code of Practice for dosimetry based on standards of absorbed dose to water *IAEA TRS*-398 (Vienna: IAEA) 

In summary, the situation is not fully clear. What seems to be clear, however, is that the report number should appear, one way or another. So I based my commit on what I read in the Vancouver style, that made use of numbers in report, and kept a "Report " prefix (was: "Report No."). This is a liberty that I took, but of course there may be better ways. 


Yes I meant Roman type, not numerals. Sorry.
Massimo

**Comment by adam3smith** [comment]
might be better to use
 
```
<group delimiter=" ">
  <text variable="genre"/>
  <text variable="number"/>
</group>
```

instead for more flexibility. That way, most of the above examples would be possible, depending on the content of genre (Type in Zotero).

**Comment by rmzelle** [comment]
@adam3smith, isn't "Technical Report Series" more akin the collection-title? (see http://aurimasv.github.io/z2csl/typeMap.xml#map-report and http://www-pub.iaea.org/mtcd/publications/pdf/trs398_scr.pdf )

**Comment by adam3smith** [comment]
I find Type vs. Series for Reports very unclear. (e.g. Technical Report Series is probably a series, but if it just said IAEA Technical Report -- would that be type???)

In citation styles where I want to be thorough, I test for one, and if absent, use the other. (APA is an example for that). I think that's probably the most robust approach? Also has the example of working with both Zotero and Mendeley field set up (Mendeley doesn't display genre/type for reports by default).

**Comment by rmzelle** [comment]
> Technical Report Series is probably a series, but if it just said IAEA Technical Report -- would that be type???

I guess it depends on whether it's just a relatively free-form description accompanying the number (e.g. "report no. 44") or an established title under which the report series are cataloged. I'd say "Technical Report Series" and "IAEA Technical Report" could both qualify as the latter.

**Comment by massimopinto** [comment]
Thank you for your interest. 
The IAEA publishes several series of reports: http://www-pub.iaea.org/books/, the TRS being just one of them. For reference, the reports are identified with TRS-Number and TECDOC-Number, to name the ones that I have come across more often. 

For completeness, I am pasting here the format that is outputted by Papers when I adopt the *Physics in Medicine and Biology* style as it is at present. Using "TRS-398" in the *number* field of my citation record (tech report):

Andreo P, Burns D T, Hohlfeld K, Huq M, Kanai T, Laitano R F, Smyth V and Vynckier S 2000 *Absorbed Dose Determination in External Beam Radiotherapy, An International Code of Practice for Dosimetry Based on Standards of Absorbed Dose to Water* (Vienna: International Atomic Energy Agency) 

which is not ideal, as TRS-398 is omitted. If I move "TRS-398" in the *Volume* field however:

Andreo P, Burns D T, Hohlfeld K, Huq M, Kanai T, Laitano R F, Smyth V and Vynckier S 2000 *Absorbed Dose Determination in External Beam Radiotherapy, An International Code of Practice for Dosimetry Based on Standards of Absorbed Dose to Water* vol TRS-398 (Vienna: International Atomic Energy Agency)

"vol" is prefixed. One alternative of course is moving TRS-398 to the title, but that has generated way too many different outputs, on the same journal. 
When you encounter these issues, do you contact the journal's editorial office?

**Comment by adam3smith** [comment]
Does Papers not have a series or a type field for tech reports? 
We sometimes do contact journals, but here the issue seems more conceptual than related to doubts of the journal style.

**Comment by massimopinto** [comment]
@adam3smith Looks like it does not, as at version 3.4.1. If it did, would that offer a solution?

**Comment by adam3smith** [comment]
I think so -- we would then do something along the lines of what I suggest above: i.e. just include both the type/series field (i.e. `genre` and/or `collection-title` for CSL purposes) together with the number for reports, and wouldn't hard-label the "Report" part, which I find too inflexible for the different types of reports users may enter into their reference manager.

**Comment by massimopinto** [comment]
I conacted the developers of Papers, who mentioned that as at the current version, Papers has still some rudimentary series support:

>If you enter the series title ("Technical Reports Series") into the subtitle field, Papers should then recognize it as the series title on export (e.g., when exporting to BibTeX)—at least for certain publication types.

We are still in touch to see what's the way to push this forward, as far as the Papers' end is concerned.
Massimo
